### PR TITLE
added polyfill message

### PR DESCRIPTION
**Pull request by edvinerikson**
Fixes https://github.com/facebook/relay/issues/690
I couldn't really find more es6 stuff other than what it checks (except for `Map` which got transpiled to `_Map` for some reason).
Not sure about tests.. Do we need a test for this and where should it go if so?
cc @josephsavona

### CODE REVIEW

**Comment by edvinerikson** [comment]
Flow install seems to have some issues.

File: src/__forks__/Relay.js [hunk 1]
```diff
@@ -16,6 +16,15 @@
 const RelayDefaultNetworkLayer = require('RelayDefaultNetworkLayer');
 const RelayPublic = require('RelayPublic');
 const RelayStore = require('RelayStore');
+const warning = require('warning');
+
+if (__DEV__) {
+  warning(
+    Promise && Array.prototype.find,
```
Comment (kassens) [inline src/__forks__/Relay.js]: maybe change this to `typeof Promise === 'function'` to not throw if it's not defined.

**Comment by kassens** [comment]
Thanks for doing this, just a small update please and we should be good to merge this.

File: src/__forks__/Relay.js (hunk 1 above)
Comment (edvinerikson) [inline src/__forks__/Relay.js]: I thought of doing `global.Promise` first but wasn't sure if it gets transformed correctly in browsers.
I'll go with the first. :+1:

**Comment by edvinerikson** [comment]
Awesome, I've updated the PR :)

**Comment by kassens** [comment]
Thanks for the update!

**Pull request closed without merging**